src/sentry_plugins/clubhouse/plugin.py

```diff
@@ -33,6 +33,9 @@ class ClubhousePlugin(CorePluginMixin, IssuePlugin2):
             IntegrationFeatures.ISSUE_BASIC,
         ),
     ]
+    deprecation_date = "Sept 20, 2021"
```

Comment:
I think it might save some headache later to make this a datetime
```python
deprecation_date = datetime.datetime(2021, 9, 20)
```
Then you can get this string in the serializer as 

```python
deprecation_date.strftime("%b %-d, %Y")
```